src/main/java/org/myrobotlab/service/IntegratedMovement.java

```diff
@@ -559,7 +559,7 @@ public void setDHLink(String arm, ServoControl servo, double d, double theta, do
     if (engines.containsKey(arm)) {
       IMEngine engine = engines.get(arm);
       DHLink dhLink = new DHLink(servo.getName(), d, r, MathUtils.degToRad(theta), MathUtils.degToRad(alpha));
-      servo.addIKServoEventListener(this);
+      // servo.addIKServoEventListener(this);

```

Comment:
This is a breaking change.  At this point, the IntegratedMovement service should subscribe to the servo events instead.  (same data is passed in ServoData)

Reply:
IM service polluted several other services in its implementation (Servo, JME, InMoov, ...)
Priority is to fix up the other services and contain the problem.
Fixing IM is lower priority.